app/src/main/res/values/strings.xml

```diff
@@ -269,10 +264,10 @@
     <!-- Feature discovery -->
     <string name="app_info_title">How The Algorithm Works</string>
     <string name="app_info_body">
-        1. You choose between two users repeatedly.\n
-        2. The app then creates a stack ranking.\n
-        3. A daily algorithm matches for mutual attraction.\n
-        4. Enjoy your top 10 matches!
+        1. Choose between two profiles repeatedly.\n
+        2. App creates profile rankings.\n
+        3. Algorithm matches for mutual attraction.\n
```

Comment:
I wonder if there is a way to edit "algorithm" out for the no well-known reasons.

"Daily update of matches with highest mutual attraction." (way too long to fit in to the UI)
"Matches daily updated by mutual attraction."

Maybe, there is something to be done by changing the fourth line as well.

"3. Matches updated daily."
"4. Matches limited to the top 10 mutual attraction matches" (way too many words)

"3. Matched by highest mutual attraction daily."
"4. Limited to the 10 highest quality matches."


Hm, maybe there is an entirely different spin by changing all bullet items:
1. Choose between two profiles repeatedly.
2. Repeated winners become your top choices.
3. Win his/her comparisons as well.
4. Only the top 10 most mutual connections get matched.
(This version doesn't sound very good. Though, you may see how by changing all four together, there may be a really strong version hiding.)